src/Microsoft.Framework.ApplicationHost2/Program.cs

```diff
@@ -29,6 +31,13 @@ public Program(IAssemblyLoaderContainer container, IApplicationEnvironment envir
 
         public Task<int> Main(string[] args)
         {
+#if DEBUG && ASPNET50
+            if(args.Length > 0 && string.Equals(args[0], "dbg", StringComparison.OrdinalIgnoreCase))
```

Comment:
We already added a --debug flag to klr.exe which will be the way to run this eventually.